modules/apigee_edge_apidocs/src/Form/ApiDocSettingsForm.php

```diff
@@ -41,13 +42,55 @@ public function getFormId() {
    * {@inheritdoc}
    */
   public function submitForm(array &$form, FormStateInterface $form_state) {
+    $entity_type = \Drupal::entityTypeManager()
+      ->getStorage('apidoc')
+      ->getEntityType();
+    $config = $this->configFactory()->getEditable('apigee_edge_apidocs.settings');
+
+    $options = $form_state->getValue('options');
+    $config->set('default_revision', (bool) $options['new_revision'])->save();
+
+    $args = [
+      '@type' => $entity_type->getLabel(),
+    ];
+    drupal_set_message($this->t('@type settings have been updated.', $args));
```

Comment:
`$this->messenger()`

Reply:
Fixed, using `$this->messenger()->addStatus()`.